wagtail/admin/views/generic/models.py

```diff
@@ -150,6 +153,13 @@ def _annotate_queryset_updated_at(self, queryset):
         )
         return queryset.annotate(_updated_at=models.Subquery(latest_log))
 
+    def get_base_queryset(self):
+        base_queryset = super().get_base_queryset()
+        if self.i18n_enabled:
```

Comment:
question: considering downstream viewset definitions may or may not include the locale column, I wonder if this should also check that the locale column is `self.columns`?

Reply:
Good point. Although I do wonder, if we access `locale_id` and use the cached mapping from the IDs to the locale names, wouldn't the prefetch be unnecessary? :thinking:

Reply:
9efca30